src/crypto/crypto_keys.h

```diff
@@ -242,7 +242,7 @@ class NativeKeyObject : public BaseObject {
     std::shared_ptr<KeyObjectData> data_;
   };
 
-  BaseObject::TransferMode GetTransferMode() const override;
+  uint32_t GetTransferMode() const override;
```

Comment:
Nit: not really a fan of changing this from an enum.

Reply:
Updated!